src/app/admin/createUser/CreateUserForm.tsx

```diff
@@ -120,8 +122,8 @@ const CreateUserForm: React.FC<{}> = () => {
                         Invite User
                     </Button>
                 )}
-
-                {submitError && <Alert severity="error">{submitError}</Alert>}
+                {serverError && <Alert severity="error">{serverError}</Alert>}
```

Comment:
Instead of directly using serverError as a string here, I think we'd want to show custom message based on serverError.

In principle, if we want to change the error message shown on the CreateUserForm when something goes wrong, we want to be editing `CreateUserForm.tsx`, not `adminInviteUser.ts`, right?

Additionally, if we were to have two separate pathways to invite a user on the website, maybe the error messages should be different for the same error (because there's a suspected user error in one case but not the other for example).

All this to say, `adminInviteUser.ts` shouldn't concern itself with error messages displayed on frontend.

Reply:
okay sure, makes sense